opencog/cython/opencog/utilities.pyx

```diff
@@ -1,5 +1,7 @@
+from contextlib import contextmanager
 from opencog.atomspace cimport AtomSpace
-from opencog.type_constructors import set_type_ctor_atomspace
+from opencog.atomspace import create_child_atomspace
+from opencog.type_constructors import get_type_ctor_atomspace, set_type_ctor_atomspace
 
```

Comment:
Someday we should rename these to get and `set_current_atomspace` since the phrase `type_ctor` is hard to remember and confusing.  And also, probably all of `opencog.type_constructors` should be merged into `opencog.atomspace`